shared/lib/timestamp-utilities.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const assert = require('matrix-public-archive-shared/lib/assert');
+const { MS_LOOKUP } = require('matrix-public-archive-shared/lib/reference-values');
+const { ONE_DAY_IN_MS, ONE_HOUR_IN_MS, ONE_MINUTE_IN_MS, ONE_SECOND_IN_MS } = MS_LOOKUP;
+
+// TODO: It would be good to add a few test for these
```

Comment:
Add some tests for these

Reply:
Added ✅ -> https://github.com/matrix-org/matrix-public-archive/pull/139/commits/2db3c4a93bb3ba7877e3ba74b3ea34f55630428a